cgroups/Cargo.toml

```diff
@@ -14,7 +14,10 @@ nix = "0.22.0"
 procfs = "0.10.1"
 log = "0.4"
 anyhow = "1.0"
-oci-spec = { git="https://github.com/utam0k/oci-spec-rs/", tag = "v0.4.0-with-bugfix"}
+# TODO: Fetch from crate.io instead of git when next release oci-spec-rs
+# ref: https://github.com/containers/oci-spec-rs/pull/69
+# oci-spec = "0.5.1"
+oci-spec = { git = "https://github.com/containers/oci-spec-rs",  rev = "5018f8e5b0355a82c08962cefa5ab07a05b930c6" }
```

Comment:
Is the current commit beyong 0.5.1 or before?

Reply:
@yihuaf No, it's a later commit than 0.5.1.
I'm waiting for the release of a version that incorporates the following patch.
https://github.com/containers/oci-spec-rs/pull/69